### DISCUSSION TITLE
Swiper Vue

### DISCUSSION THREAD

**Discussion by zapalagrzegorz** [nolimits4web/swiper#4868] [Q&A]
Is it possbile to register swiper vue globally?  Local registration is of course ok.
I wanted need to use ```is="swiper-slider``` directive on SSRendered HTML:
I tried this but see no effect to global registration.

```
import SwiperCore, { Navigation, A11y, Autoplay } from "swiper";
import { Swiper, SwiperSlide } from "swiper/vue";

SwiperCore.use(Navigation, A11y, Autoplay);

const app = createApp({
        components: {
        Swiper, 
        SwiperSlide 
    },
```

**Comment by nolimits4web** [comment]
https://learnvue.co/2020/08/how-to-register-a-vue3-global-component/
